torch/_inductor/codegen/rocm/rocm_template.py

```diff
@@ -90,15 +88,18 @@ def generate(  # type: ignore[override]
             call_args,
             expected_args,
         )
-        extra_args = V.graph.sizevars.size_hints(
-            map(sympy.expand, call_args[len(expected_args) :])
-        )
-        # create the BenchmarkRequest
+
+        size_args = (
+            self.size_args() if hasattr(self, "size_args") else ()
+        )  # subclass should define def size_args()
+        size_args_ints = [
+            V.graph.sizevars.symbolic_hint(arg) for arg in size_args
```

Comment:
I think `sizevars.size_hint` is the more preferred method here since it's more unbacked symint friendly =)